### PR TITLE
Home work 23

### PR DESCRIPTION
**Pull request by UnitOmega13**

### CODE REVIEW

File: src/src/main/java/controller/ChangeProductServlet.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.logging.Logger;
+
+@WebServlet(value = "/users/edit")
+public class ChangeProductServlet extends HttpServlet {
+
+    private static final Logger LOGGER = Logger.getLogger(String.valueOf(ChangeProductServlet.class));
+    private static final ProductService productService = ProductServiceFactory.getInstance();
+    private Product product;
```
Comment (Sofasmile) [inline src/src/main/java/controller/ChangeProductServlet.java]: try doesn't use private fields like product

**Review by Sofasmile — COMMENTED** [review]
jstl is good, make small fixes from previous review

File: src/src/main/java/controller/ChangeProductServlet.java (hunk 1 above)
Comment (UnitOmega13) [reply src/src/main/java/controller/ChangeProductServlet.java]: fixed

Comment (UnitOmega13) [reply src/src/main/java/controller/ChangeProductServlet.java]: fixed

**Pull request closed without merging**